requirements.txt

```
@@ -1,5 +1,5 @@
-aiidalab==22.02.0
-aiidalab-widgets-base==1.3.3
+aiidalab~=22.7.1
```

This must be pinned.
```suggestion
aiidalab==22.7.1
```